src/krux/display.py

```diff
@@ -317,11 +317,11 @@ def to_lines_endpos(self, text, max_lines=None):
         # Append/replace ellipsis on last line if we didn't finish the text
         if line_count == max_lines and start < len(text):
             len_last = len(lines[-1])
-            if len_last + 3 > columns:
-                lines[-1] = lines[-1][: columns - 2] + ".."
-                end += columns - len_last - 3
+            if len_last + 1 > columns:
```

Comment:
Here we don't have the same logic as before I think. Had you found a bug? I think to maintain the original it would be `len_last + 2 > columns`

Reply:
I need to give this one some thought.

I suspect that the 3 is there because previously the ellipsis was 3 "." chars, and that I forgot to shrink the condition from 3 to 2 when going to "..".

More importantly, it must be missing a test on one of the edges of "can barely fit appending ellipsis" and "too big to append ellipsis", so I'll get test(s) done to prove which one is correct, then I'll adjust or leave it as `len_last + 1` (which I suspect is correct).